docs/books/incus_server/06-profiles.md

```diff
@@ -9,31 +9,31 @@ tags:
   - incus profiles
 ---
 
-Throughout this chapter you will need to run commands as your unprivileged user ("incusadmin" if you've been following from the beginning in this book).
+Throughout this chapter, you must run commands as your unprivileged user ("incusadmin" if you've followed this book from the beginning).
 
-You get a default profile when you install Incus, and you cannot remove this profile or modify it. That said, you can use the default profile to create new profiles to use with your containers.
+When you install Incus, you get a default profile, which you cannot remove or modify. You can use the default profile to create new profiles for your containers.
 
-If you examine your container listing, you will notice that the IP address in each case is from the bridged interface. In a production environment, you may want to use something else. This might be a DHCP assigned address from your LAN interface or even a statically assigned address from your WAN.
+If you examine your container listing, you will notice that the IP address in each case is from the bridged interface. In a production environment, you may want to use something else. This might be a DHCP-assigned address from your LAN interface or a statically assigned address from your WAN.
 
-If you configure your Incus server with two interfaces and assign each an IP on your WAN and LAN, it is possible to assign your container's IP addresses based on the interface the container needs to be facing.
+If you configure your Incus server with two interfaces and assign each an IP on your WAN and LAN, you can assign your container's IP addresses based on the interface the container needs to be facing.
 
-As of version 9.4 of Rocky Linux (and really any bug for bug copy of Red Hat Enterprise Linux) the method for assigning IP addresses statically or dynamically with the profiles does not work.
+As of Rocky Linux version 9.4 (and any bug for bug copy of Red Hat Enterprise Linux), the method for assigning IP addresses statically or dynamically with the profiles does not work.
 
-There are ways to get around this, but it is annoying. This appears to have something to do with changes made to Network Manager that affect `macvlan`. `macvlan` allows you to create many interfaces with different Layer 2 addresses.
+There are ways to get around this, but it isn't very pleasant. This appears to have something to do with changes made to Network Manager that affect `macvlan`. `macvlan` allows you to create many interfaces with different Layer 2 addresses.
 
```

Comment:
"isn't very pleasant" introduces a conjunction, which we do not want. Perhaps "is not very pleasant."